activation/activation.go

```diff
@@ -638,8 +639,10 @@ func (b *Builder) createAtx(
 	challenge *types.NIPostChallenge,
 ) (*types.ActivationTx, error) {
 	pubEpoch := challenge.PublishEpoch
+	// TODO: in future, encode the right NiPoST challenge version depending on the pubEpoch.
+	challengeHash := wire.NIPostChallengeToWireV1(challenge).Hash()
```

Comment:
There might be a transitional period where the builder needs to now both versions but after that switch passed we can remove the logic to build a V1 ATX. 🙂